sphere.ts

```diff
@@ -118,22 +118,22 @@ namespace space {
      * @param orientation the axis on which the sphere will build: either X, Y, or Z
      * @param useBlockOrder each slice of the sphere will use the order of the blocklist if true or random if false
      */
-    //% blockId=space_sphereRandomLayerBlocks
-    //% block="sphere of random layers from $blockList|center $center|radius $radius|around $orientation|in order of blocks $useBlockOrder|$operator"
+    //% blockId=space_sphereCustomLayerBlocks
```

Comment:
this is fine because nobody is using this extension yet, but never change block ids! this will break all existing projects that are using this block